firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteSchema.java

```diff
@@ -691,11 +691,12 @@ private void createOverlays() {
           db.execSQL(
               "CREATE TABLE document_overlays ("
                   + "uid TEXT, "
+                  + "collection_group TEXT, "
                   + "collection_path TEXT, "
                   + "document_id TEXT, "
                   + "largest_batch_id INTEGER, "
                   + "overlay_mutation BLOB, "
-                  + "PRIMARY KEY (uid, collection_path, document_id))");
+                  + "PRIMARY KEY (uid, collection_group, collection_path, document_id))");
```

Comment:
Please don't add to primary key. We will not be able to drop the column.

Reply:
Created a separate index instead.